intermittent failure for moin.macros._tests.test_DateTime.test_Macro

https://github.com/moinwiki/moin/actions/runs/4962917141/jobs/8881573754

```
================================== FAILURES ===================================
_________________________________ test_Macro __________________________________

    def test_Macro():
        """Test: DateTime.Macro """
        flaskg.user.valid = True  # show_time creates ISO 8601 dates if user is not logged in
        macro_obj = Macro()
        # when arguments is None
        result = macro_obj.macro('content', None, 'page_url', 'alternative')
        # get the current time
        test_time = time.time()
        test_time = format_date_time(datetime.utcfromtimestamp(test_time))
>       assert test_time == result
E       AssertionError: assert 'May 12, 2023...01:07\u202fPM' == 'May 12, 2023...01:06\u202fPM'
E         - May 12, 2023, 9:01:06\u202fPM
E         ?                     ^
E         + May 12, 2023, 9:01:07\u202fPM
E         ?                     ^

src\moin\macros\_tests\test_DateTime.py:29: AssertionError
```

--- a/src/moin/macros/_tests/test_DateTime.py
+++ b/src/moin/macros/_tests/test_DateTime.py
@@ -25,8 +25,9 @@ def test_Macro():
     result = macro_obj.macro('content', None, 'page_url', 'alternative')
     # get the current time
     test_time = time.time()
-    test_time = format_date_time(datetime.utcfromtimestamp(test_time))
-    assert test_time == result
+    test_times = [test_time, test_time - 1]  # in case our call to time.time happened just after the second rolled over
+    test_times = [format_date_time(datetime.utcfromtimestamp(t)) for t in test_times]
+    assert result in test_times
 
     arguments = ['2023-08-07T11:11:11', 'argument2']
     result = macro_obj.macro('content', arguments, 'page_url', 'alternative')

